app/admin/users.rb

```diff
@@ -1,5 +1,5 @@
 ActiveAdmin.register User do
-  permit_params :first_name, :last_name, :email, :password
+  permit_params :first_name, :last_name, :email, :password, :password_confirmation
```

Comment:
It looks like `:password_confirmation` isn't used anywhere. Maybe it can be removed?